packages/orbit-components/src/tmp-tailwind/InputGroup/index.tsx

```diff
@@ -0,0 +1,197 @@
+            {label}
+          </FormLabel>
+        )}
+
+        <div
+          className={cx(
+            "text-normal h-form-box-normal duration-fast rounded-large tb:rounded-normal z-10 w-full transition-all ease-in-out",
+            active && "outline-blue-normal outline outline-2",
+            disabled ? "bg-form-element-disabled-background" : "bg-form-element-background",
+            !errorReal && "shadow-form-element",
+            !errorReal && !disabled && "hover:shadow-form-element-hover",
+            Boolean(errorReal) && "shadow-form-element-error",
+            Boolean(errorReal) && !disabled && "hover:shadow-form-element-error-hover",
+          )}
+        >
+          <div className="relative flex" onBlur={handleBlurGroup}>
+            {React.Children.toArray(children).map((child, key) => {
+              // TODO: next cleanup iteration just remove this whole `flex` prop thing
+              // and cloning elements, and make children take care of their sizing themselves
+
+              const childFlex = (
+                Array.isArray(flex) && flex.length !== 1 ? flex[key] ?? flex[0] : flex
+              ) as string | undefined;
+
+              const item = child as React.ReactElement<Props>;
+              return (
+                <div
+                  key={randomId(String(key))}
+                  className={cx(
+                    "orbit-input-group-child pe-xs [&_.orbit-select-container]:after:duration-fast [&_.orbit-input-field-container]:after:duration-fast last:p-0 [&_.orbit-input-field-container]:after:absolute [&_.orbit-input-field-container]:after:end-0 [&_.orbit-input-field-container]:after:top-[50%] [&_.orbit-input-field-container]:after:z-10 [&_.orbit-input-field-container]:after:block [&_.orbit-input-field-container]:after:h-[24px] [&_.orbit-input-field-container]:after:-translate-y-[50%] [&_.orbit-input-field-container]:after:border-r [&_.orbit-input-field-container]:after:transition-colors [&_.orbit-input-field-container]:after:ease-in-out [&_.orbit-input-field-container]:after:content-['_'] [&_.orbit-input-field-container]:last-of-type:after:content-none [&_.orbit-input-field-fake-input]:hidden [&_.orbit-input-field-input~.orbit-input-field-fake-input]:shadow-none [&_.orbit-select-container>select]:bg-transparent [&_.orbit-select-container>select]:shadow-none [&_.orbit-select-container>select]:focus:outline-none [&_.orbit-select-container]:bg-transparent [&_.orbit-select-container]:after:absolute [&_.orbit-select-container]:after:end-0 [&_.orbit-select-container]:after:top-[50%] [&_.orbit-select-container]:after:z-20 [&_.orbit-select-container]:after:block [&_.orbit-select-container]:after:h-[24px] [&_.orbit-select-container]:after:-translate-y-[50%] [&_.orbit-select-container]:after:border-r [&_.orbit-select-container]:after:transition-colors [&_.orbit-select-container]:after:ease-in-out [&_.orbit-select-container]:after:content-['_'] [&_.orbit-select-container]:last-of-type:after:content-none",
```

Comment:
`[&_.orbit-input-field-container]:after:top-[50%]` can be `[&_.orbit-input-field-container]:after:top-1/2`, to avoid custom class. Same for the other selector.

Reply:
adjusted, and split to 3 lines 👍